Makefile

```diff
@@ -48,11 +50,22 @@ tarballs:
 docker:
 	@docker build -t prometheus:$(shell git rev-parse --short HEAD) .
 
+rpm-nosystemd:
+	@echo ">> building rpm package for no-systemd distros"
+	@rpmbuild --buildroot "$(PWD)/.build/nosystemd" -D "_topdir $(PWD)" -D "_builddir $(PWD)/.build/nosystemd" -D "src_root $(PWD)" -D "version $(VERSION)" -D 'use_systemd 0' -bb prometheus.spec
+	@mv RPMS/*/*.rpm "$(PWD)"/
+
+rpm-systemd:
+	@echo ">> building rpm package for systemd distros"
+	@rpmbuild --buildroot "$(PWD)/.build/systemd" -D "_topdir $(PWD)" -D "_builddir $(PWD)/.build/systemd" -D "src_root $(PWD)" -D "version $(VERSION)" -D 'use_systemd 1' -bb prometheus.spec
+	@mv RPMS/*/*.rpm "$(PWD)"/
+
+rpm:	build rpm-systemd rpm-nosystemd
```

Comment:
We include binary in package with ldd depends only from libc:
```
linux-vdso.so.1 =>  (0x00007ffd2429d000)
libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f75ffcde000)
libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f75ff914000)
/lib64/ld-linux-x86-64.so.2 (0x000055a6eaafa000)
```


So we have almost nothing in dependencies but init-system, and we can create universal package for all systemd distros and another one for all nosystemd distros.